### ISSUE TITLE
Windows color resource picker

### ISSUE THREAD

**Issue by ermau** [xamarin/Xamarin.PropertyEditing#172]
This is the color resources tab in the solid color picker:
![image](https://user-images.githubusercontent.com/156582/35946539-34aa11ce-0c32-11e8-953f-7a6be7e4d5df.png)

It gets reused in the gradient picker when we do that as well.

**Comment by lewing** [comment]
Shouldn't this be closed?

**Comment by ermau** [comment]
@lewing No, it's not the same as the brush resource selector. It requires some re-working of brushes before it can be done, so I've unassigned you.
